app/controllers/api/v1/classifications_controller.rb

```diff
@@ -64,9 +64,13 @@ def build_resource_for_create(create_params)
   end
 
   def lifecycle(action, classification)
-    ClassificationLifecycle.queue(classification, action)
+    if Panoptes.flipper[:cls_lifecycle_queue].enabled?
```

Comment:
could this be named a little more clearly? makes it easier for users to understand it implications of the feature from the UI without needing to look at the code. maybe `classification_lifecycle_in_background` or invert it and do `classification_lifecycle_in_request`